client/webserver/site/src/js/markets.ts

```diff
@@ -795,6 +796,7 @@ export default class MarketsPage extends BasePage {
 
   setCandleDurBttns () {
     const { page, market } = this
+    if (!market?.dex) return
```

Comment:
The only place `setCandleDurBttns` is used is in `setMarket` right after we've populated `this.market`. This is an impossible condition.